printer_mods/Mckurt/Switchwire Y Belt Tensioner/Readme.md

```diff
@@ -0,0 +1,24 @@
+| Creator | Config title | Description | V0 | V1.8 | V2.4 | VSW |
+| MCKURT11 | Switchwire Y Belt Tensioner | :x: | :x: | :x: | :x: | :heavy_check_mark: |
```

Comment:
Please remove these two lines, instead add a link to your mod to `printer_mods/README.md`.